README.md

```diff
@@ -34,6 +34,7 @@ HiClass is an open-source Python library for hierarchical classification compati
 - **[Hierarchical metrics](https://hiclass.readthedocs.io/en/latest/api/utilities.html#hierarchical-metrics):** HiClass supports the computation of hierarchical precision, recall and f-score, which are more appropriate for hierarchical data than traditional metrics.
 - **[Compatible with pickle](https://hiclass.readthedocs.io/en/latest/auto_examples/plot_model_persistence.html):** Easily store trained models on disk for future use.
 - **[BERT sklearn](https://hiclass.readthedocs.io/en/latest/auto_examples/plot_bert.html):** Compatible with the library [BERT sklearn](https://github.com/charles9n/bert-sklearn).
+- **[Hierarchical Explanability]():**  HiClass allows explaining hierarchical models using the [SHAP](https://github.com/shap/shap) package.
```

Comment:
```suggestion
- **[Hierarchical Explanability](TODO: add link here):**  HiClass allows explaining hierarchical models using the [SHAP](https://github.com/shap/shap) package.
```
It is still missing the hyperlink to read the docs

Reply:
Resolved by https://github.com/scikit-learn-contrib/hiclass/commit/af7e83abbebd90e14fbc9abf380d4c68eada4e92.